src/Core/src/Platform/Android/ImageSourcePartExtensions.cs

```diff
@@ -70,6 +70,11 @@ internal static class ImageSourcePartExtensions
 			}
 			catch (Exception ex)
 			{
+				if (setImage is not null)
+				{
+					setImage(null);
```

Comment:
Not a biggy, but setImage?.Invoke maybe? Does the same thing so take it or leave it, makes no diff :)